pkgs/development/libraries/libfprint-thinkpad/default.nix

```diff
@@ -0,0 +1,64 @@
+{ stdenv
+, fetchFromGitHub
+, pkgconfig
+, meson
+, ninja
+, libusb
+, pixman
+, glib
+, nss
+, gtk3
+, coreutils
+, gtk-doc
+, docbook_xsl
+, docbook_xml_dtd_43
+, openssl
+}:
+
+stdenv.mkDerivation rec {
+  pname = "libfprint-thinkpad";
+  version = "1.0";
+
+  src = fetchFromGitHub {
+    owner = "3v1n0";
+    repo = "libfprint";
+    rev = "2e2e3821717e9042e93a995bdbd3d00f2df0be9c";
+    sha256 = "1vps1wrp7hskf13f7jrv0dwry2fcid76x2w463wplngp63cj7b3b";
+  };
+
+  nativeBuildInputs = [
+    pkgconfig
+    meson
+    ninja
+    gtk-doc
+    docbook_xsl
+    docbook_xml_dtd_43
+  ];
+
+  buildInputs = [
+    libusb
```

Comment:
This should be `libusb1` and not libusb in compat-mode with 0.1.5 ABI [as per upsteam spec](https://github.com/3v1n0/libfprint/blob/vfs0090/meson.build#L45)
( cc @cizra )

Reply:
Thanks, fixed it with last push.